doc/rados/operations/pg-repair.rst

```diff
@@ -1,4 +1,53 @@
+============================
 Repairing PG inconsistencies
 ============================
+Sometimes a placement group might become "inconsistent". To return the
+placement group to an active+clean state, you must first determine which 
+of the placement groups has become inconsistent and then run the "pg
+repair" command on it. This page contains commands for diagnosing placement
+groups and the command for repairing placement groups that have become
+inconsistent.
+
+.. highlight:: console
+
+Commands for Diagnosing Placement-group Problems
+================================================
+The commands in this section provide various ways of diagnosing broken placement groups.
+
+The following command provides a high-level (low detail) overview of the health of the ceph cluster::
+
+  [node] # ceph health detail
+
+The following command provides more detail on the status of the placement groups::
+
+  [node] # sudo ceph pg dump --format=json-pretty
```

Comment:
to be consistent none of these should use sudo -- if it's needed for one of these commands, it'll be needed for all of them. Same for the pg repair commands

Reply:
Good catch.